src/acquisition/covid_hosp/state_daily/README.md

```diff
@@ -0,0 +1,10 @@
+# COVID-19 Reported Patient Impact and Hospital Capacity by State
```

Comment:
```suggestion
# COVID-19 Reported Patient Impact and Hospital Capacity by State Daily Snapshots
```

Reply:
"COVID-19 Reported Patient Impact and Hospital Capacity by State" is the title of the dataset per https://healthdata.gov/dataset/covid-19-reported-patient-impact-and-hospital-capacity-state

if you still want "Daily Snapshots", maybe put it in parens? "COVID-19 Reported Patient Impact and Hospital Capacity by State (Daily Snapshot)"

Reply:
Updated to add parenthesis